### ISSUE TITLE
can arm linux support conda and use conda to install linux app?

### ISSUE THREAD

**Issue by ZihaoXingstudy** [conda/conda#9279]
can arm linux support conda and use conda to install linux app?

**Comment by jjhelmus** [comment]
Yes conda can be used on Linux distributions running on Arm hardware.  `conda-forge` has a number of packages for 64-bit Arm, `linux-aarch64`.  For 32-bit Arm you can try [berryconda](https://github.com/jjhelmus/berryconda) although it targets Raspberry Pi's running Raspbian and may not work on other systems.
